apps/vr-tests/src/stories/Calendar.stories.tsx

```diff
@@ -33,4 +33,23 @@ storiesOf('Calendar - No Month Option', module)
     <Fabric>
       <Calendar value={date} showGoToToday={false} showMonthPickerAsOverlay={true} />
     </Fabric>
+  ))
+  .addStory('Show Week selection type', () => (
+    <Fabric>
+      <Calendar value={date} dateRangeType={DateRangeType.Week} />
+    </Fabric>
+  ))
+  .addStory('Show Month selection type', () => (
```

Comment:
Is this one supposed to have nothing selected?
https://screener.io/v2/states/Skgr32fUL.microsoft-fluentui/lorejoh12%2FimproveCalendarSelectionBorder/1024x768/Chrome/calendar-no-month-option-show-month-selection-type-default-0
![image](https://user-images.githubusercontent.com/5864305/128268231-78d87971-3ebb-4639-a953-c2610699240a.png)

Reply:
yes! It used to have the whole month highlighted but the designers hated it and made it so that it only highlights on hover. It's the only view type to look "blank", which is why I added the screener for it